src/arch/mipsel/trap.rs

```diff
@@ -11,7 +13,10 @@ global_asm!(include_str!("trap.S"));
 /// You **MUST NOT** modify these registers later.
 pub unsafe fn init() {
     // Set cp0 ebase(15, 1) register to trap entry
-    llvm_asm!("mtc0 $0, $$15, 1":: "r"(trap_entry as usize) :: "volatile");
+    asm!(
+        "mtc0 {trap_entry}, $15, 1",
+        trap_entry = in(reg) trap_entry,
+    );
```

Comment:
Is this correct?

Reply:
It seems right to me, but I'm not 100% sure. The MIPS target is least used, so maybe we can revisit it in the future.